vllm/v1/engine/core.py

```diff
@@ -230,7 +238,7 @@ def run_busy_loop(self):
                         # Break out the loop so we can log_stats in step().
                         if self.log_stats:
                             break
-                    except BaseException:
```

Comment:
note: do not use `BaseException`. This catches this like `SystemExit` and `asyncio.CancelledError`. We always want to catch `Exception`